documentation/autoinstall-reference.md

```diff
@@ -1,5 +1,3 @@
-# Automated Server Installs Config File Reference
```

Comment:
Is this an intended change?

Reply:
Yes.  This change was done on Discourse to drop a redundant title being shown on the page, so it should be replicated here.